src/h5web/toolbar/controls/DomainSlider/BoundEditor.tsx

```diff
@@ -0,0 +1,85 @@
+import { ReactElement, useEffect, useRef, useState } from 'react';
+import { FiCheck, FiSlash } from 'react-icons/fi';
+import { formatValue } from '../../../utils';
+import styles from './BoundEditor.module.css';
+
+interface Props {
+  label: string;
+  value: number;
+  isEditing: boolean;
+  hasError: boolean;
+  onEditToggle: (force: boolean) => void;
+  onChange: (val: number) => void;
+}
+
+function BoundEditor(props: Props): ReactElement {
+  const { label, value, isEditing, hasError, onEditToggle, onChange } = props;
+
+  const id = `${label}-bound`;
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const [inputValue, setInputValue] = useState(formatValue(value));
+
+  useEffect(() => {
+    setInputValue(isEditing ? value.toString() : formatValue(value));
+  }, [isEditing, value, setInputValue]);
+
+  useEffect(() => {
+    if (!isEditing) {
+      // Remove focus from min field when editing is turned off
+      inputRef.current?.blur();
+    }
+
+    if (isEditing && label === 'Min') {
+      // Give focus to min field when opening tooltip in edit mode
+      inputRef.current?.focus();
+    }
+  }, [isEditing, label]);
+
+  return (
+    <form
+      className={styles.boundEditor}
+      data-error={hasError || undefined}
+      data-editing={isEditing}
+      onSubmit={(evt) => {
+        evt.preventDefault();
+        onChange(Number.parseFloat(inputValue));
+        onEditToggle(false);
+      }}
+    >
+      <label id={`${id}-label`} className={styles.label} htmlFor={id}>
+        <abbr title={value.toString()}>{label}</abbr>
+      </label>
+
+      <input
+        id={id}
+        ref={inputRef}
+        className={styles.value}
+        type="text"
+        name="bound"
+        value={inputValue}
+        aria-labelledby={`${id}-label`} // https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/718
```

Comment:
Isn't it rather https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/566 ?

Reply:
haha yes, thanks, forgot to update the link 😄